src/include/daos/common.h

```diff
@@ -104,12 +104,17 @@ char *DP_UUID(const void *uuid);
 #define DP_CONT(puuid, cuuid)	DP_UUID(puuid), DP_UUID(cuuid)
 #define DF_CONTF		DF_UUIDF"/"DF_UUIDF
 
+#ifndef DAOS_BUILD_RELEASE
```

Comment:
I generally prefer ifdef when there's an else clause, and only ifndef if used without and else.

Reply:
I did this because there are 2 kinds of builds covered by this dev and debug. So it logically is if not release (meaning the other two).

Reply:
I guess I can swap it though. I dont think there will be any harm in doing that.

Reply:
Swapped and pushed a new version up.